FlightsWebsite/src/js/weather.js

```diff
@@ -63,4 +88,54 @@ function showColumnGraph(title, titleY, titleX, data) {
         }]
     });
     chart.render();
+}
+
+function showScatterGraph(title, titleY, data) {
```

Comment:
it is a bit more complicated for this case .. but I want to use it also in another use case so I tried to make it already for one or more origins obtained in HTTP response